tests/expected/decode/test.gif.stdout

```diff
@@ -45,7 +45,7 @@
 │       │              0   0   0
 │       │              0   0   0
 │       ├── blocks <- repeat gif.block :=
-│       │   └── 0 <- gif.block :=
+│       │   └── 0 :=
```

Comment:
Losing the format here seems mildly disadvantageous, would it be possible to keep it for the multiline case, given that the rest of the line is free anyway, and only omit it for the atomic value case?

Reply:
I added a perhaps-precarious condition under which the omission will not occur (format is ItemVar, Apply, or Dynamic, and the value-fragment spans multiple lines). It seems to do the trick, but might be a bit messy to maintain or have edge-cases we might not yet have coverage for.